ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.13.yaml

```diff
@@ -80,12 +80,12 @@ tests:
   steps:
     cluster_profile: vsphere-multizone
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones
-- as: installer-rehearse-nutanix-ipi-disconnected
+- as: installer-rehearse-nutanix-ipi-proxy
```

Comment:
I suggest to not update the test name frequently, let use keep it like `installer-rehearse-nutanix-dis`, then you do some testing in the future, you just need to update workflow, no need to run `make jobs` every time.